docusaurus/docs/schema/0-Schema.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Overture Maps Schema
+slug: /schema
+
+# This page is available at docs.overturemaps.org/schema
+---
+import CodeBlock from '@theme/CodeBlock';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import JSONSchemaViewer from "@theme/JSONSchemaViewer";
+import generateResolverOptions from "@site/src/components/shared-libs/generateResolverOptions"
+import StringifyObject from "@site/src/components/shared-libs/stringifyObject" 
+import yamlLoad from "@site/src/components/yamlLoad" 
+import BuildingFeature from "!!raw-loader!@site/docs/_examples/buildings/empire-state-building.yaml";
+import MainDefs from "!!raw-loader!@site/docs/_schema/defs.yaml";
+
+Overture Maps data is structured by three components: the schema, the data model, and GERS. The schema describes the shape of the data and devises the constraints applied to that data. The data model specifies what types of features exist, their geometries, how the features relate to each other, and what kind of attributes they have. Every feature in Overture Maps has a unique ID, and for some features types that unique ID is part of the Global Entity Reference System, a system of encoding map data to a shared universal reference. You can see all three components in this description of one feature, the Empire State Building:
+
+ <CodeBlock language="json">{ JSON.stringify(yamlLoad(BuildingFeature), null, 2) }</CodeBlock>
```

Comment:
@jenningsanderson I'm not sure this is the correct way to reference that yaml file

Reply:
Yep! This is fixed now and the file is JSON